constants.py

```diff
@@ -73,7 +73,7 @@ class ServoExtension(Enum):
 
 
 # Don't log more than x packets for StandbyState and LandedState
-LOG_CAPACITY_AT_STANDBY = 5000  # This is equal to (x/2 + x = 3x/2 = 5000 => x = 3333 = 3.33 secs of data)
+IDLE_LOG_CAPACITY = 5000  # This is equal to (x/2 + x = 3x/2 = 5000 => x = 3333 = 3.33 secs of data)
```

Comment:
I like showing the math in the comment, it has me thinking though, maybe it would be more helpful to make this a number that evenly corresponds with a certain amount of seconds? E.g. 4 seconds so 6000 packets?

Alternatively, make a different constant for amount of seconds we want the buffer to be for and then have it just do the math.

Idk just ideas, isn't really that important either way

Reply:
we can just make a function which converts the seconds to packets and then we can call it from here. I don't want another constant though. But I'm not going to do that in this PR